recipe/meta.yaml

```diff
@@ -31,39 +31,46 @@ requirements:
   run:
     - python
     - setuptools
```

Comment:
Is setuptools a runtime dependency still? I think we have solved the version issue I think we added this for?

Reply:
I don't think so looking at `setup.cfg`? I've just removed it from here, lets see if it builds fine.

Reply:
Looks 👍 without